app/models/user_profile.rb

```diff
@@ -1,28 +1,28 @@
 class UserProfile < ActiveRecord::Base
-ratyrate_rateable "ratings"
-belongs_to :user
-after_initialize :default_values
- validates :first_name,      
-           :last_name,
-           :gender,
-           :mobile_no,
-           :address,
-           :city,
-           :pincode, presence: true
-  
-validates_format_of :first_name,:last_name,:city, :with => /[-a-z]+/
-has_attached_file :avatar, 
+  ratyrate_rateable "ratings"
+  belongs_to :user
+  after_initialize :default_values
+  validates :first_name,      
+            :last_name,
+            :gender,
+            :mobile_no,
+            :address,
+            :city,
+            :pincode, presence: true
+  validates_format_of :first_name,:last_name,:city, :with => /[-a-z]+/
+  has_attached_file :avatar, 
 	                        :path => ":rails_root/public/system/:attachment/:id/:style/:filename",
 	                        :url => "/system/:attachment/:id/:style/:filename",
 	                        :default_url => ":rails_root/assets/avatar/default.jpg",
                           :styles => { :small => "30x15>", :medium => "155x85>" }
-validates :avatar,presence: true
-validates_attachment_size :avatar, :less_than => 5.megabytes
-validates_attachment_content_type :avatar, 
+  validates :avatar,presence: true
+  validates_attachment_size :avatar, :less_than => 5.megabytes
```

Comment:
User hash through out like `less_than: 5.megabytes`. Please change in all related code